docs/UserGuide.md

```diff
@@ -259,182 +277,191 @@ Format: `find PREFIX KEYWORD [KEYWORD]…​`
 
-Format: `untag INDEX t/TAGNAME`
+Format: `untag PERSON_NUMBER t/TAG_NAME`
 
 * Untag a specified contact based on their `INDEX` with a `Tag`.
+* `TAG_NAME` is case-insensitive.
 * The index refers to the index number shown in the displayed person list.
 * The index **must be a positive integer** 1, 2, 3, …​.
 
-### Deleting a tag : `delete-tag`
+### Deleting a tag : `delete-tag` or `dtag`
 
 Deletes a `Tag` from WedLinker.
 
-Format: `delete-tag t/TAGNAME [f/]`
+Format: `delete-tag t/TAG_NAME [f/]` or `dtag t/TAG_NAME [f/]`
 
 * Deletes a `Tag` from WedLinker.
+* `TAG_NAME` is case-insensitive.
 * The `Tag` must exists in WedLinker.
 * The `Tag` cannot be assigned to any contacts.
 * If the `Tag` is in used, you can use `f/` to force the deletion of the `Tag` and unassign this tag from all contacts.
 
 ## Wedding Features
 
-### Adding a Wedding : `create-wedding`
+### Adding a Wedding : `create-wedding` or `cw`
 
 Creates a `Wedding` within WedLinker to be with contacts.
 
-Format: `create-wedding w/WEDDINGNAME`
+Format: `create-wedding w/WEDDING_NAME` or `cw w/WEDDING_NAME`
 
-* The `WEDDINGNAME` is alphanumeric and can contain whitespaces.
+* The `WEDDING_NAME` should only contain alphanumeric characters, spaces or the following characters: / . , ' & : ( )
+* `WEDDING_NAME` is case-insensitive.
 * Weddings are unique in WedLinker, there would not be any duplicated Weddings.
-* Contacts can be assigned to the Wedding using the [assign-wedding](#assign-contact-to-a-wedding--assign-wedding) command.
+* Contacts can be assigned to the Wedding using the [assign-wedding](#assigning-contact-to-a-wedding-assign-wedding) command.
 
-### Assign contact to a Wedding : `assign-wedding`
+### Assigning contact to a Wedding : `assign-wedding` or `asw`
 
 Assigns a contact to a `Wedding`.
 
-Format: `assign-wedding INDEX w/WEDDINGNAME`
+Format: `assign-wedding PERSON_NUMBER w/WEDDING_NAME` or `asw PERSON_NUMBER w/WEDDING_NAME`
 
-* Assigns a specified contact to the `Wedding` based on their `INDEX`.
+* Assigns a specified contact to the `Wedding` based on their `PERSON_NUMBER`.
+* `WEDDING_NAME` is case-insensitive.
 * The index refers to the index number shown in the displayed person list.
 * The index **must be a positive integer** 1, 2, 3, …​.
 * The `Wedding` must exists in WedLinker before it can be assigned.
 * If the `Wedding` does not exist, you can use `f/` to force the creation and assignment of the `Wedding`.
 
-### Edit Wedding details : `edit-wedding`
+### Editing Wedding details : `edit-wedding` or `ew`
 
 Edits the details of a `Wedding`.
 
-Format: `edit-wedding INDEX [w/WEDDINGNAME] [a/ADDRESS]`
+Format: `edit-wedding INDEX [w/WEDDING_NAME] [a/ADDRESS]` or `ew INDEX [w/WEDDING_NAME] [a/ADDRESS]`
 
-* Edits the specific `Wedding` at the INDEX when in [list-wedding](#listing-all-weddings-list-weddings) view.
+* Edits the specific `Wedding` at the INDEX when in [list-weddings](#listing-all-weddings-list-weddings) view.
```

Comment:
I'm noting this here - not sure if it's directly related to this PR, but INDEX is still being used here - not sure if this is something that someone else is standardising already